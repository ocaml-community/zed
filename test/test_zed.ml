let () =
  let s = "\247\165\165\165" in
  let (ofs, _, message) = Zed_utf8.next_error s 0 in
  Printf.printf "next_error (scalar value too large) = (%d, _, %S)\n" ofs message
