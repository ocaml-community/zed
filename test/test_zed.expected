next_error (scalar value too large) = (0, _, "scalar value too large in UTF8 sequence")
